src/commands/synthetics/utils.ts

```diff
@@ -534,3 +534,22 @@ export const retry = async <T, E extends Error>(
 
   return trier()
 }
+
+export const parseVariablesFromCli = (variableArguments: string[] = []): {[key: string]: string} | undefined => {
+  const variables: {[key: string]: string} = {}
+
+  for (const variableArgument of variableArguments) {
+    const separatorIndex = variableArgument.indexOf('=')
+
+    if (separatorIndex === -1) {
+      continue
```

Comment:
Should we add a warning on ignored invalid variables?
Also small edge case I can think of is something like `=val` setting an empty variable name